_data/sidebars/home_sidebar.yml

```diff
@@ -12,8 +12,9 @@ entries:
       url: /
       output: web
       type: homepage
-    - title: Documentation
-      url: /docs/
+      output: web
```

Comment:
Actually I just noticed that this is a dupe of line 13 and can be removed.

Reply:
Good catch. Thank you!